app/lib/usecase/optics/create_case.rb

```diff
@@ -0,0 +1,18 @@
+module Usecase
+  module Optics
+    class CreateCase
+      def initialize(optics_gateway:, presenter:, get_bearer_token:)
+        @optics_gateway = optics_gateway
+        @presenter = presenter
+        @get_bearer_token = get_bearer_token
```

Comment:
Aware this requests a token on each request.
Caching strategy to be discussed.

Reply:
i think this is a non-issue for now